pkg/labelsfilter/filter.go

```diff
@@ -131,9 +148,24 @@ func ParseLabelPrefixCfg(prefixes []string, file string) error {
 		cfg.LabelPrefixes = append(cfg.LabelPrefixes, p)
 	}
 
+	if fromCustomFile {
+		found := false
+		for _, label := range cfg.LabelPrefixes {
+			if label.Source+":"+label.Prefix == reservedLabelsPattern {
+				found = true
+				break
+			}
+		}
+
+		if !found {
+			log.Warnf("'%s' is not included in the final label list, which may result to "+
+				"traffic interruptions when label list changed", reservedLabelsPattern)
```

Comment:
I think we should error out here instead:
```suggestion
			log.Errorf("'%s' needs to be included in the final label list for " +
				"Cilium to work properly.", reservedLabelsPattern)
```